### ISSUE TITLE
option for using different thousands separator

### ISSUE THREAD

**Issue by tpapp** [JuliaIO/Formatting.jl#45]
The default is `,`. Supporting arbitrary chars, eg `_` which is valid read syntax in Julia, would be nice.

**Comment by ScottPJones** [comment]
Yes, it would also be nice to be able to correctly handle European formatting of numbers, using `,` instead of `.` for the decimal point, and `.` as the thousands separator.
I'll see about adding those features to `JuliaString/Format.jl`, and then see about backporting to here.

**Comment by jakewilliami** [comment]
@ScottPJones I know this is old, but any updates on this?  I think a lot of people would find this useful.

**Comment by ScottPJones** [comment]
@jakewilliami This has been fixed in [JuliaString/Format.jl](https://github.com/JuliaString/Format.jl)
